src/main/java/org/mifos/connector/ams/paygops/camel/route/PaygopsRouteBuilder.java

```diff
@@ -88,10 +104,9 @@ public void configure() {
                     String body = exchange.getIn().getBody(String.class);
                     JSONObject jsonObject = new JSONObject(body);
                     Integer errorCode = jsonObject.getInt("error");
-                    String errorDescription = jsonObject.getString("error_message");
-                    exchange.setProperty(ERROR_CODE, errorCode);
-                    exchange.setProperty(ERROR_INFORMATION, jsonObject.toString(1));
-                    exchange.setProperty(ERROR_DESCRIPTION, errorDescription);
+                    String errorDescription   = jsonObject.getString("error_message");
+                    String errorInfo = jsonObject.toString(1);
```

Comment:
we should have an error dictionary to mention all these error codes, we cannot randomly hardcode numbers like this. it should have a proper Enum